### ISSUE TITLE
Can't get libconfig to link properly into my VS2015 windows projects

### ISSUE THREAD

**Issue by kyranf** [hyperrealm/libconfig#148]
I've been trying in vain for hours now to work out what is wrong with my project setup for including/linking to libconfig++. 

Here's my super quick test project which also fails like my much larger project does, using the same paths and properties for the project. 

![image](https://user-images.githubusercontent.com/7900774/62624843-e6444480-b91b-11e9-8e86-0ab3d2c69942.png)

I have set up the paths as required so the compiler an find the .lib , I have successfully built AND RUN the examples for libconfig++, but for the life of me I cannot work out the proper way to set up my VS project to include and link properly with the library. 

From some of the other issues i found on here, I tried adding the #define LIBCONFIG_STATIC before including the header, but that doesn't work. Also, note I don't mind linking a DLL at runtime, I just need the functionality and I'm not fussed about if it's static linking to compile into the .exe or as a .DLL. 

please help me :( 

edit: I should note that i'm using Visual studio 2015, Windows 10, and I'm trying to compile x64 (does this even matter??)

**Comment by kyranf** [comment]
I just changed to compile in x86 configuration, added the paths and libconfig++.lib  etc, and it actually worked. So this is a matter of a .lib/.dll compiled in x86  rather than x64. 

How can I compile the libconfig++ library in x64? I have the Visual Studio projects for them, are there any recommendations for x64?

**Comment by kyranf** [comment]
I went to the libconfig_vs2008 solution,  added a new build configuration of x64 for the libconfig++ project (only),  and said "yes" to copy x86 configuration settings. 

I then built only libconfig++ project (with x64 configuration).

I then went to my configtest project and changed the library include directory to  point at ./x64/Release/   instead ( I added the extra folder of x64 ) 

I then recompiled my test project in x64 configuration, and it worked!  I hope this helps others who get caught out like I did...
